packages/@ember/-internals/runtime/lib/type-of.js

```diff
@@ -77,7 +77,7 @@ const { toString } = Object.prototype;
 
   @method typeOf
   @for @ember/utils
-  @param {Object} item the item to check
+  @param item the item to check
```

Comment:
what's the type if the type annotation is removed? `any`?

Reply:
I believe so.

Reply:
It's either `any` or `unknown`.